Lib/Rabbit4000/tcpip/DMAETH100.LIB

```diff
@@ -808,34 +817,33 @@ dma_rcv_isr::
    jr    nz,.reset_rx_port ; 5
 
    ; Check if only status rcv'd with no packet data
+   ; This is the catch all for errors with R6000
 	ld		hl,(ix+[_decb]+rcvlen)
    test	hl
    jr		z,.test_rx_error
 
    ; Check for valid Rcv Packet
-	ld		a,(ix+[_decb]+rcvlsbstat)
+   ; Don't know where this is documented for R6000?

```

Comment:
Not the greatest, but `dmaeth100_prt_nicreg()` seems to have a description for the 16-bit receive status.  Bit 7 of the LSB corresponds to "Receive OK", so I think we should leave this in for the 6000 as well as the 5000.